src/components/segmentedControl/index.tsx

```diff
@@ -92,11 +92,11 @@ const SegmentedControl = (props: SegmentedControlProps) => {
     containerStyle,
     style,
     segments,
-    activeColor = Colors.primary,
+    activeColor = Colors.$textPrimary,
```

Comment:
`activeColor` is used in both text and icon, currently they are the same but I think this should be reflected in the code.